bcda/bcdacli/cli_test.go

```diff
@@ -958,12 +958,12 @@ func (s *CLITestSuite) TestCloneCCLFZips() {
 }
 
 func (s *CLITestSuite) TestGenerateAlrData() {
-	initialCount := postgrestest.GetALRCount(s.T(), s.db, "A9994")
-	args := []string{"bcda", "generate-synthetic-alr-data", "--cms-id", "A9994",
+	initialCount := postgrestest.GetALRCount(s.T(), s.db, "A9990")
```

Comment:
A9994 is used extensively and in detail throughout unit-test for ALR, such as metaKey ID. To prevent clash, mixing up the ACOs.